src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -278,4 +284,6 @@ public void upsert(TransactionEntity transaction) {
                 .bindBean(transaction)
                 .execute());
     }
+
+
```

Comment:
can these empty line breaks be removed please

Reply:
They can.

Reply:
Updated.